ts/.npmrc

```diff
@@ -1,3 +1 @@
 registy=https://registry.npmjs.org/
-
-always-auth=true
```

Comment:
`always-auth` was originally added when this project was using an internal Azure DevOps registry. It is not needed now when using the public npmjs registry. I'm removing it because the auth was failing in online builds after updating to Node.js 18.

Reply:
The problem was references to the internal feed still in `package-lock.json`. I rebuilt `package-lock.json` using only the public feed.